webapp-django/crashstats/signature/static/signature/js/signature_report.js

```diff
@@ -157,18 +157,16 @@ SignatureReport.init = function () {
         });
 
         // Show or hide filters.
-        searchSection.on('click', '.toggle-filters', function (e) {
+        $('.toggle-filters').click(function (e) {
```

Comment:
Ideally you should use the `.on('click'` instead. It's just a nit but I remember reading that all these hardcoded events lick `.click`, `.submit` are discouraged in favor of `.on('click', ` and `.on('submit', `